apps/postybirb-ui/src/website-components/bluesky/bluesky-login-view.tsx

```diff
@@ -0,0 +1,147 @@
+import { Trans } from '@lingui/macro';
+import { Box, Button, Stack, TextInput } from '@mantine/core';
+import { notifications } from '@mantine/notifications';
+import { BlueskyAccountData, BlueskyOAuthRoutes } from '@postybirb/types';
+import React, { useState } from 'react';
+import websitesApi from '../../api/websites.api';
+import { ExternalLink } from '../../components/external-link/external-link';
+import HttpErrorResponse from '../../models/http-error-response';
+import { LoginComponentProps } from '../../models/login-component-props';
+
+const formId = 'bluesky-login-form';
+
+export default function BlueskyLoginView(
+  props: LoginComponentProps<BlueskyAccountData>,
+): JSX.Element {
+  const { account } = props;
+  const { data, id } = account;
+  const [username, setUsername] = useState(data?.username ?? '');
+  const [password, setPassword] = useState(data?.password ?? '');
+  const [isSubmitting, setSubmitting] = useState<boolean>(false);
+
+  const usernameValidationErrors: React.ReactNode[] = [];
+  if (username.startsWith('@')) {
+    usernameValidationErrors.push(
+      <Trans comment="Bluesky login form">
+        You don't need to input the @. Unless your username{' '}
+        <strong>really</strong> contains it
+      </Trans>,
+    );
+  }
+  if (username && !username.includes('.')) {
+    usernameValidationErrors.push(
+      <Trans comment="Bluesky login form">
+        Be sure that the username is in the format handle.bsky.social. Or if you
```

Comment:
q: bsky allows other domains?

Reply:
yes, for example https://bsky.app/profile/fenrir784.ru